waiter/src/waiter/scheduler/shell.clj

```diff
@@ -321,24 +330,45 @@
     (assoc instance :healthy? (perform-health-check instance health-check-url http-client))
     instance))
 
+(defn- alive?
+  "Returns true if the process is running.
+   It first tries to query the process handle, which is absent we fallback to the shell."
```

Comment:
I think there is a word missing somewhere in here:

> which is absent we fallback to the shell

Reply:
Updated docstring.